src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs

```diff
@@ -361,5 +362,93 @@ private static FixAllSuggestedAction GetFixAllSuggestedAction(IEnumerable<Sugges
 
         public void MessageBox(string message)
             => ExecuteOnActiveView(view =>  System.Windows.MessageBox.Show(message));
+
+        public void VerifyDialog(string dialogAutomationId, bool isOpen)
+        {
```

Comment:
Do we need to guarantee that this happens on the UI thread, or is it OK for it to happen on a background thread?